USERS.md

```diff
@@ -620,6 +620,12 @@ Users (Alphabetically)
       L: https://sealos.io
       Q: @fanux, @yangchuansheng
 
+    * N: SeatGeek
+      D: SeatGeek uses Cilium as the default CNI for AWS hosted clusters
+      U: CNI, ClusterMesh, Network Policy, Hubble
+      L: https://seatgeek.com
+      Q: @byxorna
+
```

Comment:
we would love to have this in separate PR

Reply:
Good suggestion, I opened https://github.com/cilium/cilium/pull/34603 👍🏽